src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/TidligereVedtaksperioderInfo.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { IFinnesTidligereVedtaksperioder, ITidligereVedtaksperioder } from './typer';
+import { GridTabell } from '../../../Felles/Visningskomponenter/GridTabell';
+import TabellVisning, { TabellIkon } from '../Tabell/TabellVisning';
+
+function mapTrueFalse(bool: boolean): string {
+    return bool ? 'Ja' : 'Nei';
+}
+
+const FinnesTidligereVedtaksperioder: FC<{
+    finnesTidligereVedtaksperioder: IFinnesTidligereVedtaksperioder;
+}> = ({ finnesTidligereVedtaksperioder }) => {
+    const { overgangsstønad, barnetilsyn, skolepenger } = finnesTidligereVedtaksperioder;
+    return (
+        <GridTabell kolonner={3}>
+            <>
+                <TabellVisning
+                    ikon={TabellIkon.REGISTER}
+                    tittel="Har bruker historikk i Infotrygd"
+                    verdier={[
+                        { stønad: 'Overgangsstønad', verdi: overgangsstønad },
+                        { stønad: 'Barnetilsyn', verdi: barnetilsyn },
+                        { stønad: 'Skolepenger', verdi: skolepenger },
+                    ]}
+                    kolonner={[
+                        {
+                            overskrift: 'Stønad',
+                            tekstVerdi: (d) => d.stønad,
+                        },
+                        {
+                            overskrift: 'Historikk i Infotrygd',
```

Comment:
Blir denne litt "smør på flesk" ? Eller tenker vi å vise `Historikk i ny løsning` i en tabell under senere?

Reply:
Enig, det var litt for å gjøre det veldig tydelig att tabellen gjelder `Infotrygd`, Lars like egentligen forslaget uten noen header for den andre kolonnen, men att det var litt tydligere med den. 

Det er mye mulig det blir visning av data fra ny løsning senere